resources/views/layouts/app.blade.php

```diff
@@ -6,15 +6,33 @@
         <meta name="csrf-token" content="{{ csrf_token() }}">
 
         <title>{{ config('app.name', '') }}</title>
-
-        <link href="{{ asset('css/core.css') }}" rel="stylesheet" type="text/css">
     </head>
     <body>
         @yield('content')
 
         @if (app()->environment() == 'local')
+            <script>
+                window.stylePaths = {
+                    app: "{{ mix('css/app.css') }}",
+                    theme: {
+                        darkTweaks: "{{ mix('css/dark-theme.custom.css') }}",
+                        dark: "{{ mix('css/dark-theme.css') }}",
+                        light: "{{ mix('css/core.css') }}",
+                    },
+                };
+            </script>
             <script src="{{ mix('js/app.js') }}"></script>
         @else
+            <script>
+                window.stylePaths = {
+                    app: "{{ asset('css/app.css') }}",
+                    theme: {
+                        darkmods: "{{ asset('css/dark-theme.custom.css') }}",
+                        dark: "{{ asset('css/dark-theme.css') }}",
+                        light: "{{ asset('css/core.css') }}",
+                    },
+                };
+            </script>
```

Comment:
Preferably add a helper that auto-detects whether `mix` or `asset()` should be called.